packages/ui-extensions/src/surfaces/point-of-sale/api/smartgrid-api/smartgrid-api.ts

```diff
@@ -1,3 +1,7 @@
+
+/** * 
+ * @deprecated Use `ActionApiContent` instead.
```

Comment:
```suggestion
 * @deprecated Use `ActionAPI` instead.
```

Just changing the wording here, since from the interface perspective, partners are going to see "actionAPI". ActionAPIContent is just the type 😄